hist/histv7/inc/ROOT/RHistData.hxx

```diff
@@ -161,6 +161,32 @@ public:
       for (size_t b = 0; b < fOverflowBinContent.size(); ++b)
          fOverflowBinContent[b] += other.fOverflowBinContent[b];
    }
+
+   /// Multiply by a scalar.
+   template <typename T>
+   void MultiplyByScalar(const T scalar) {
+      for (size_t b = 0; b < fBinContent.size(); ++b)
+         fBinContent[b] *= scalar;
+      for (size_t b = 0; b < fOverflowBinContent.size(); ++b)
+         fOverflowBinContent[b] *= scalar;
+   }
+
+   /// Divide by an other RHistStatContent, assuming same bin configuration.
+   void Divide(const RHistStatContent& other) {
+      assert(fBinContent.size() == other.fBinContent.size());
+      assert(fOverflowBinContent.size() == other.fOverflowBinContent.size());
+      for (size_t b = 0; b < fBinContent.size(); ++b)
+         fBinContent[b] /= other.fBinContent[b];
+      for (size_t b = 0; b < fOverflowBinContent.size(); ++b)
+         fOverflowBinContent[b] /= other.fOverflowBinContent[b];
+      fEntries += other.fEntries;
```

Comment:
It is not immediately obvious to me that this is the right thing to do from a mathematical point of view.